src/main/java/liquibase/nosql/executor/NoSqlExecutor.java

```diff
@@ -181,8 +183,22 @@ public void execute(final UpdateStatement updateStatement) throws DatabaseExcept
 
     @Override
     public void execute(final SqlStatement sql) throws DatabaseException {
-        disableRowAffectedMessage();
-        this.execute(sql, emptyList());
+        Map<String, Object> scopeValues = new HashMap<>();
+        scopeValues.put(JdbcExecutor.ROWS_AFFECTED_SCOPE_KEY, new AtomicInteger(0));
```

Comment:
If we invoke execute multiple times we'll be always resetting it. Is it expected or should we just add a new `AtomicInteger(0)` if it's not present in the scope yet?

Reply:
I changed to use `putIfAbsent` and now it will only be included if it doesn't exist. It's working.